components/toast_wrapper/index.tsx

```diff
@@ -2,37 +2,60 @@
 // See LICENSE.txt for license information.
 
 import {connect} from 'react-redux';
-import {bindActionCreators} from 'redux';
-import {withRouter} from 'react-router-dom';
+import {bindActionCreators, Dispatch} from 'redux';
+import {withRouter, RouteComponentProps} from 'react-router-dom';
+
+import {IDMappedObjects} from '@mattermost/types/utilities';
+
+import {Post} from 'packages/types/src/posts';
+
+import {GlobalState as ToastWrapperState} from '@mattermost/types/store';
+
+import {ViewsState} from 'types/store/views';
```

Comment:
nit picking, please rearrange the props how we are doing in `components/emoji_picker/emoji_picker.tsx` for consistency